docs/howto/operate/add-hub.md

```diff
@@ -1,8 +1,67 @@
 # Add a new hub
 
-Adding a new hub requires that we do three things:
+## Step 1: Decide where the hub will live
 
-1. [Create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new).
-2. Create the hub via appending a new hub entry in the appropriate cluster file under
-   `config/hubs` file (see [](../../topic/config.md) for more information).
-3. Start following [team process](team-process) around operating the hubs.
+Hubs need to exist on a kubernetes cluster, so we will need to
+decide which cluster it should live in.
+
+1. For standalone hubs where we are being paid or the client has cloud
+   credits, we will need to create a cluster specifically for this hub
+   before deploying it.
+2. For hubs we are running for free, they will go into the `pilot-hubs`
+   cluster.
+3. Hubs run with the support of [cloudbank](https://www.cloudbank.org/),
+   will go in the `cloudbank` cluster.
+
```

Comment:
I like this categorization!

Reply:
+1!

Reply:
I wonder if, rather than hard-coding "cloudbank" here, we should keep a list of "special" clusters like the CloudBank one (basically any cluster that has more than a 1-1 mapping onto customers) and then reference that table?

I seem to remember us having an issue about having such a "source of truth" for clusters, but maybe this was just for hubs?

Reply:
It was for hubs, indeed (IIRC).
I am +1 about having something more dynamic here for "special" clusters...

Reply:
Although maybe we can add that later... when we have more than one "special" cluster?